core/src/apps/misc/get_firmware.py

```diff
@@ -0,0 +1,46 @@
+from micropython import const
+from typing import TYPE_CHECKING
+
+from trezor import utils, wire, workflow
+from trezor.messages import FirmwareChunk, FirmwareChunkAck, GetFirmware, Success
+from trezor.ui.layouts import confirm_action, draw_simple_text
+
+from .get_firmware_hash import _render_progress
+
+if TYPE_CHECKING:
+    from trezor.wire import Context
+
+CHUNK_SIZE = const(1024 * 4)
+PROGRESS_TOTAL = const(13 * 128 * 1024)
```

Comment:
preferrably 13 -> `utils.FIRMWARE_SECTORS_COUNT`

Reply:
done in [f1f4abe](https://github.com/trezor/trezor-firmware/pull/2248/commits/f1f4abe9811e527e8d179106bdaf2dac4f6cc275)

it might break `const()` but that is probably not important